api/server/router/local/image.go

```diff
@@ -319,6 +319,7 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 	buildConfig.ForceRemove = httputils.BoolValue(r, "forcerm")
 	buildConfig.MemorySwap = httputils.Int64ValueOrZero(r, "memswap")
 	buildConfig.Memory = httputils.Int64ValueOrZero(r, "memory")
+	buildConfig.ShmSize = r.FormValue("shmsize")
```

Comment:
This should be consistent with `"memory"`, hence use `httputils.Int64ValueOrZero`.